ramp-for-gutenberg.php

```diff
@@ -12,6 +12,10 @@
 
 // This file loads Ramp, and modifies behaviors for Gutenberg on VIP Go
 
+if ( defined( 'VIP_GO_DISABLE_RAMP' ) && VIP_GO_DISABLE_RAMP ) {
```

Comment:
A strong comparisson yoda condition would keep this in alignment with VIP standards ( `&& true === VIP_GO_DISABLE_RAMP` )

Reply:
Yes please.